src/widgetastic_patternfly/__init__.py

```diff
@@ -122,7 +122,7 @@ def active(self):
 
     @property
     def disabled(self):
-        return self.browser.get_attribute('disabled', self) == 'disabled'
+        return 'disabled' in self.browser.classes(self)
```

Comment:
Shouldn't this check both of these things then? Why are you removing the old conditions? I believe it should be `or`ed

Reply:
okay will make the changes accordingly.

Reply:
This was the reason I tried to write a new button, totally forgot that I can use 'or'